tdrs-backend/tdpservice/data_files/validators.py

```diff
@@ -46,11 +46,11 @@ def validate_file_extension(file_name: str):
         raise ValidationError(msg)
 
 
-def validate_file_infection(file):
+def validate_file_infection(file, uploaded_by):
     """Validate file is not infected by scanning with ClamAV."""
     av_client = ClamAVClient()
     try:
-        is_file_clean = av_client.scan_file(file, file.name)
+        is_file_clean = av_client.scan_file(file, uploaded_by)
```

Comment:
We also need to track the user that uploaded the file or we won't have anything to link the `LogEntry` to.